src/test/resources/authz.json

```diff
@@ -25,16 +26,29 @@
         "admin",
         "api",
         "security",
-        "m2m"
+        "m2m",
+        "support"
       ],
       "route": "getAttributeByName"
     },
+    {
+      "verb": "GET",
+      "roles": [
+        "admin",
+        "api",
+        "security",
+        "m2m",
+        "support"
+      ],
+      "route": "getAttributes"
+    },
```

Comment:
Of course this change is not enough, you should add the test that uses this json value in `AttributeApiServiceAuthzSpec.scala`

Reply:
Sorry for the misunderstanding, next time I pay more attention